src/bin/zmcontrol

```diff
@@ -216,7 +211,7 @@ sub doStatus {
     my $status = 0;
     print "Host $localHostName\n";
     foreach ( sort keys %{$services} ) {
-        if ( $_ eq "syncshare" ) { next; }
+        if ( $_ eq "directory-server" || $_ eq "zimlet" || $_ eq "zimbraAdmin" || $_ eq "zimbra" ) { next; }
```

Comment:
Why directory-server is skipped?

Reply:
Yes, we should also consider extracting this logic into a separate subroutine to avoid this kind of mistakes?
or maybe create an array of blacklisted services and do something like this `$service ~~ @array` inside these if's

Reply:
Could be, but there's not much time left in order to let it go in the next RC, so I think we can open a `technical improvement` ticket an do it for the next sprint